deploy/template.yaml

```diff
@@ -775,8 +775,14 @@ Resources:
       ServiceNamespace: ecs
       StepScalingPolicyConfiguration:
         AdjustmentType: PercentChangeInCapacity
-        Cooldown: 420
+        Cooldown: 30
```

Comment:
I'm wondering what the benefit of being this aggressive with the scale in is, I would have thought we'd want to wait longer for previous scaling activity to take effect before scaling down again?

I know this is from the ADR but just comparing against [what core have implemented](https://github.com/govuk-one-login/ipv-core-front/blob/main/deploy/template.yaml#L652)

Reply:
That sounds sensible to me 👍